Detectors/Raw/src/RawFileReaderWorkflow.cxx

```diff
@@ -160,6 +161,8 @@ class rawReaderSpecs : public o2f::Task
                  hbOrbExpected, hbOrbRead, hdrTmpl.splitPayloadIndex, tfID,
                  link.origin.as<std::string>(), link.description.as<std::string>(), link.subspec);
           }
+          hdrTmpl.firstTForbit = hbOrbRead + loopsDone * nhbexp; // for next parts
+          reinterpret_cast<o2::header::DataHeader*>(hdMessage->GetData())->firstTForbit = hdrTmpl.firstTForbit;
```

Comment:
The actual assignment in the header message needs to be outside the conditional branch. This sets firstTForbit only for the first of the split parts (while keeping the value for the others in hdrTmpl but never setting it)

Reply:
I don't think there is bug: for every part (i.e. HBF) the header is created as a clone of the hdrTmpl
https://github.com/AliceO2Group/AliceO2/blob/d13b16c729e08126060a16c1b03c79834369b8bc/Detectors/Raw/src/RawFileReaderWorkflow.cxx#L144
So, I need to set explicitly only the firstTForbit for hdrTmpl and the very 1st part of the TF multipart (since its header is created before we get access to the 1st RDH)

Reply:
thanks, indeed, I was looking for exactly that.